TUTORIAL.md

```diff
@@ -17,7 +17,7 @@ contracts.ruby brings code contracts to the Ruby language. Code contracts allow
 A simple example:
 
 ```ruby
-Contract Contracts::Num, Contracts::Num => Contracts::Num
+Contract Contracts::Builtin::Num, Contracts::Builtin::Num => Contracts::Builtin::Num
```

Comment:
If I think about `validator` list, it does not need to have `Builtin::` in it too.